### PR TITLE
added button component

### PR DESCRIPTION
**Pull request by Geljinskiy**

### CODE REVIEW

File: src/components/common/MainButton/MianButton.module.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@import '../../../styles/colors';
```
Comment (myroslavsuprun) [inline src/components/common/MainButton/MianButton.module.scss:1]: Omit all the dots
just `styles/colors`

File: src/components/common/MainButton/MainButton.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import styles from './MianButton.module.scss';
+
+const MainButton = ({ btnStyle, children }) => {
+  return (
+    <button className={`${styles.button} ${styles[btnStyle]}`}>
```
Comment (myroslavsuprun) [inline src/components/common/MainButton/MainButton.jsx:5]: Add type for the button.

Make it possible to pass through props, but let the default value to be "button"

` ({ btnStyle, children, type = 'button' }) 
type={type}
`

**Review by myroslavsuprun — CHANGES_REQUESTED** [review]
LGTM, 

correct what I asked.

File: src/components/common/MainButton/MainButton.jsx (hunk 2 above)
Comment (Geljinskiy) [reply src/components/common/MainButton/MainButton.jsx]: I think, prop types wont be redundant either)

**Review by myroslavsuprun — APPROVED** [review]

**Pull request merged**